radix-engine-interface/src/time/instant.rs

```diff
@@ -0,0 +1,63 @@
+use crate::time::constants::*;
+use sbor::*;
+
+#[derive(Encode, Decode, TypeId, Copy, Clone, Debug, Eq, PartialEq)]
+pub struct Instant {
+    pub seconds_since_unix_epoch: i64,
```

Comment:
This file is mostly unchanged, but gh fails to show a proper diff (I've moved it) :(
the changes are:
* u64 -> i64 for seconds_since_unix_epoch
* and added `checked_mul` in add_xyz methods